care/facility/api/serializers/patient_otp.py

```diff
@@ -41,13 +41,15 @@ def create(self, validated_data):
         otp = rand_pass(settings.OTP_LENGTH)
 
         if settings.USE_SMS:
-            send_sms(
-                otp_obj.phone_number,
-                (
+            message = sms.TextMessage(
+                content=(
                     f"Open Healthcare Network Patient Management System Login, OTP is {otp} . "
                     "Please do not share this Confidential Login Token with anyone else"
                 ),
+                recipients=[otp_obj.phone_number],
             )
+            connection = sms.initialize_backend()
```

Comment:
Lets not have a separate initialize, let the class do it if needed. we dont need to explicitly state it